torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -135,4 +135,20 @@ xla::XlaOp BuildDropout(const xla::XlaOp& input, float probability) {
   return input * mask;
 }
 
+std::vector<xla::XlaOp> CreateBroadcastTensors(
+    tensorflow::gtl::ArraySlice<const xla::XlaOp> operands) {
+  XLA_CHECK(!operands.empty()) << "broadcast_tensors cannot take an empty list";
```

Comment:
This is an API, which means it can be called from many contexts. Remove/replace broadcast_tensors or, much better since we test outside, remove the message.

Reply:
Done.